draft-ietf-rats-corim.md

```diff
@@ -539,6 +541,41 @@ Described in {{sec-common-validity}}.
 {::include cddl/unprotected-corim-header-map.cddl}
 ~~~
 
+## Signer authority of securely conveyed unsigned CoRIM {#sec-conveyed-signer}
+
+An unsigned (#6.501-tagged) CoRIM may be a payload in an enveloping signed document.
```

Comment:
Is line 546 referring to the CoRIM bundle use case ? If yes, could you please add a reference to the CoRIM Bundle subsection as we have two cases, listed here but explanation spreaded across

Reply:
the bundle is described in a subsection a few sentences later. Does it need a link?